dataline-acceptance-tests/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins {
```

Comment:
This project should still be built through the regular build, just not run

Reply:
This is the current behavior